### ISSUE TITLE
[BUG] Video info dialog loses focus after trailer

### ISSUE THREAD

**Issue by Bequiet19** [jurialmunkey/skin.arctic.horizon.2#747]
### Skin section

Video Info Dialog

### Current Behavior

I noticed a small bug. When I enter in video info dialog and play the trailer, after the finished trailer it seems to buttons lose focus. The first click on left,right... any button nothing happens. After second click, the navigation works again. If I stop the trailer with the back button or with up or down, the focus is there. Also, if the trailer doesn't exist, the same situation occurs. I hope I explained it well.

### Expected Behavior

-

### Steps To Reproduce

-

### Screenshots and Additional Info

-

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Not much I can do about it unfortunately. There's no "onplaybackend" type of event for skins, so there's no way for the skin to send focus back from the trailer window to the control you previously were on. The only event trigger available to the skin is a direction press or ok/back click (so when you do click it can then send you back but not until that happens).
